Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -165,6 +218,67 @@ def set_default_job_id(client: Client, args: Dict[str, Any]) -> CommandResults:
 ''' MAIN FUNCTION '''
 
 
+def send_event_feedback(client: Client, feature_mapping_field: str, args: Dict[str, Any]) -> CommandResults:
+    job_id = args.get("job_id", None)
+    if not job_id:
+        job_id = client.get_default_job_id()
+    event_id = args.get("event_id")
+    mappings = parse_mappings(feature_mapping_field)
+
+    username = args.get("username")
+    label = args.get("label")
+    closing_notes = args.get("closing_notes", "")
+    indicators = args.get("indicators", None)
+    arcanna_label = mappings.get(label, None)
+    if arcanna_label is None:
+        raise Exception(f"Error in arcanna-send-feedback.Wrong label={label}")
+
+    response = client.send_feedback(job_id, event_id, username, arcanna_label, closing_notes, indicators)
+    readable_output = f' ## Arcanna send event feedback results: {response}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Arcanna.Event',
+        outputs_key_field='feedback_status',
+        outputs=response
+    )
+
+
+def send_bulk_events(client: Client, feature_mapping_field: str, args: Dict[str, Any]) -> dict:
+    job_id = args.get("job_id")
+    events = argToList(args.get("events"))
+    mappings = parse_mappings(feature_mapping_field)
+    mapped_events = []
+    for event in events:
+        closing_status = event.get("closingReason")
+        closing_notes = event.get("closeNotes")
+        closing_user = event.get("closeUser")
+        arcanna_label = mappings.get(closing_status)
+        title = event.get("name")
+        severity = event.get("severity")
+
+        body = client.map_to_arcanna_raw_event(job_id, event, severity, title)
+        body["label"] = client.map_to_arcanna_label(arcanna_label, closing_notes, closing_user)
+
+        mapped_events.append(body)
+
+    return client.send_bulk(job_id, mapped_events)
```

Comment:
This command does not return a `CommandResults` object but returns a json. Please change it to use CommandResults with the `outputs` field. After that, please update the outputs in the yml file. 
If you don't want to return anything, please remove the return statement.